openbb_terminal/parent_classes.py

```diff
@@ -1485,6 +1485,13 @@ def call_load(self, other_args: List[str]):
             default=False,
             help="Show performance information.",
         )
+        parser.add_argument(
+            "--india",
+            dest="india",
+            action="store_true",
+            default=False,
+            help="Show indian stocks.",
```

Comment:
this help needs to be more descriptive.  Will only work where ns is the proper suffix and not every source.

Reply:
updated